domain-ee/ee-ep-merge-app/integration/conftest.py

```diff
@@ -0,0 +1,50 @@
+import pytest_asyncio
+from integration.mq_endpoint import MqEndpoint
+from src.python_src.config import QUEUES, REPLY_QUEUES, ClientName
+
+
+@pytest_asyncio.fixture(autouse=True, scope="session")
+def get_claim_contentions_endpoint():
+    name = ClientName.GET_CLAIM_CONTENTIONS
+    endpoint = MqEndpoint(name, QUEUES[name], REPLY_QUEUES[name])
+    return endpoint
+
+
+@pytest_asyncio.fixture(autouse=True, scope="session")
+async def put_tsoj_endpoint():
+    name = ClientName.PUT_TSOJ
+    endpoint = MqEndpoint(name, QUEUES[name], REPLY_QUEUES[name])
+    return endpoint
+
+
+@pytest_asyncio.fixture(autouse=True, scope="session")
+async def update_claim_contentions_endpoint():
+    name = ClientName.UPDATE_CLAIM_CONTENTIONS
+    endpoint = MqEndpoint(name, QUEUES[name], REPLY_QUEUES[name])
+    return endpoint
+
+
+@pytest_asyncio.fixture(autouse=True, scope="session")
+async def cancel_claim_endpoint():
+    name = ClientName.CANCEL_CLAIM
+    endpoint = MqEndpoint(name, QUEUES[name], REPLY_QUEUES[name])
+    return endpoint
```

Comment:
These four methods are all pretty similar. Maybe this could just be something like `return helper_func(ClientName.CANCEL_CLAIM)`?